recipes/linuxbridge.rb

```diff
@@ -54,3 +54,9 @@
 end
 
 include_recipe 'openstack-network::ml2_linuxbridge'
+
+# When iptables-ng creates new rules, restart linuxbridge-agent so that it generates the iptables rules after iptables
+# is restarted.
+edit_resource(:service, 'neutron-plugin-linuxbridge-agent') do
+  subscribes :restart, 'ruby_block[create_rules]'
```

Comment:
This references the resource found [here](https://github.com/chr4-cookbooks/iptables-ng/blob/v2.2.8/recipes/manage.rb#L24-L36).